### ISSUE TITLE
PHP Fatal Error in RemoteWebDriver::findElement()

### ISSUE THREAD

**Issue by bchecketts** [php-webdriver/php-webdriver#944]
### Bug description

RemoteWebDriver::execute() may return a null value.  When this happens from `findElement()`, it causes `JsonWireCompat::getElement()` to throw a PHP Fatal error

```
PHP Fatal error:  Uncaught TypeError: Argument 1 passed to Facebook\WebDriver\Remote\JsonWireCompat::getElement() must be of the type array, null given, called in <myPath>/vendor/php-webdriver/webdriver/lib/Remote/RemoteWebDriver.php on line 215 and defined in <myPath>/api/vendor/php-webdriver/webdriver/lib/Remote/JsonWireCompat.php:21
Stack trace:
#0 <myPath>/vendor/php-webdriver/webdriver/lib/Remote/RemoteWebDriver.php(215): Facebook\WebDriver\Remote\JsonWireCompat::getElement()
#1 <myfile>.php(410): Facebook\WebDriver\Remote\RemoteWebDriver->findElement()
```

### How could the issue be reproduced
I wish I could capture some details about the page content, but the PHP Fatal error causes the session to die immediately, so I can't see what is happening.

* Php-webdriver version: 1.11.0
* PHP version:  7.2.24
* Browser driver (chromedriver/geckodriver...) version:  95.0.4638.69
* Browser used + version: 
* Operating system:

**Comment by andreaschiffo** [comment]
Similar issue here

`ACME\QAAutomation\CaptchaTest::testCaptchaOnPDP
TypeError: Argument 1 passed to Facebook\WebDriver\Remote\JsonWireCompat::getElement() must be of the type array, string given, called in /home/aschiffo/Documents/Andrea/ACME/ACME-qa-automation/vendor/php-webdriver/webdriver/lib/Remote/RemoteWebDriver.php on line 211`

@OndraM what I do is
```
const CLOSE_NEWSLETTER_WIDGET_CLASS = "close-widget-newsletter";
$this->waitForClass(self::CLOSE_NEWSLETTER_WIDGET_CLASS, true)->click();
```
which eventually ends up to _RemoteWebDriver::findElement_
`return $this->newElement(JsonWireCompat::getElement($raw_element));`
where _$raw_element_ is _"/session//element_"

It might be that the element I'm looking for is not preset but I'd expect a NoSuchElementException or similar. 

IMPORTANT NOTE
This happens to me only if I run tests with standalone chromedriver
`java -jar -Dwebdriver.chrome.driver="./vendor/selenium/chromedriver" -jar ./vendor/selenium/selenium-server-4.1.0.jar standalone`
but if I run chromedriver in a grid via docker compose as [here](https://github.com/SeleniumHQ/docker-selenium/blob/trunk/docker-compose-v3.yml), I do not encounter this issue and the test runs fine.

**Comment by andreaschiffo** [comment]
> Similar issue here
> 
> `ACME\QAAutomation\CaptchaTest::testCaptchaOnPDP TypeError: Argument 1 passed to Facebook\WebDriver\Remote\JsonWireCompat::getElement() must be of the type array, string given, called in /home/aschiffo/Documents/Andrea/ACME/ACME-qa-automation/vendor/php-webdriver/webdriver/lib/Remote/RemoteWebDriver.php on line 211`
> 
> @OndraM what I do is
> 
> ```
> const CLOSE_NEWSLETTER_WIDGET_CLASS = "close-widget-newsletter";
> $this->waitForClass(self::CLOSE_NEWSLETTER_WIDGET_CLASS, true)->click();
> ```
> 
> which eventually ends up to _RemoteWebDriver::findElement_ `return $this->newElement(JsonWireCompat::getElement($raw_element));` where _$raw_element_ is _"/session//element_"
> 
> It might be that the element I'm looking for is not preset but I'd expect a NoSuchElementException or similar.
> 
> IMPORTANT NOTE This happens to me only if I run tests with standalone chromedriver `java -jar -Dwebdriver.chrome.driver="./vendor/selenium/chromedriver" -jar ./vendor/selenium/selenium-server-4.1.0.jar standalone` but if I run chromedriver in a grid via docker compose as [here](https://github.com/SeleniumHQ/docker-selenium/blob/trunk/docker-compose-v3.yml), I do not encounter this issue and the test runs fine.

Resolved updating to latest chromedriver version

**Comment by Mike-Benoit** [comment]
Is this strictly a bug in the Chrome Driver, as I'm seeing this same issue with the latest Chrome/Chrome WebDriver v100.0.4896.6

```
PHP Fatal error:  Uncaught TypeError: Argument 1 passed to Facebook\WebDriver\Remote\JsonWireCompat::getElement() must be of the type array, string given, called in .../vendor/php-webdriver/webdriver/lib/Remote/RemoteWebDriver.php on line 211 and defined in .../vendor/php-webdriver/webdriver/lib/Remote/JsonWireCompat.php:21
Stack trace:
#0 .../vendor/php-webdriver/webdriver/lib/Remote/RemoteWebDriver.php(211): Facebook\WebDriver\Remote\JsonWireCompat::getElement()
#1 .../vendor/php-webdriver/webdriver/lib/WebDriverExpectedCondition.php(139): Facebook\WebDriver\Remote\RemoteWebDriver->findElement()
#2 [internal function]: Facebook\WebDriver\WebDriverExpectedCondition::Facebook\WebDriver\{closure}()
#3 ../vendor/php-webdriver/webdriver/lib/WebDriverWait.php(54): call_user_func()
#4 myapp.php(1608): Facebook\WebDriver\WebDriverWait->until()
```

**Comment by OndraM** [comment]
FYI, this is "fixed" by #1001 . Instead of fatal error, exception is thrown.
